conmon-rs/server/src/rpc.rs

```diff
@@ -12,8 +15,50 @@ impl conmon::Server for Server {
         _: conmon::VersionParams,
         mut results: conmon::VersionResults,
     ) -> Promise<(), capnp::Error> {
-        info!("Got a request");
+        debug!("Got a request");
         results.get().init_response().set_version(VERSION);
         Promise::ok(())
     }
+
+    fn create_container(
+        &mut self,
+        params: conmon::CreateContainerParams,
+        mut results: conmon::CreateContainerResults,
+    ) -> Promise<(), capnp::Error> {
+        let req = pry!(pry!(params.get()).get_request());
+        debug!(
+            "Got a create container request for id {}",
+            pry!(req.get_id())
+        );
+        let args = match self.generate_runtime_args(&params) {
+            Ok(x) => x,
+            Err(e) => return Promise::err(e),
+        };
+
+        let mut child = match Command::new(self.config().runtime()).args(args).spawn() {
+            Ok(x) => x,
+            Err(e) => return Promise::err(e.into()),
+        };
```

Comment:
```suggestion
        let mut child = pry!(Command::new(self.config().runtime()).args(args).spawn());
```

Reply:
goodness I tried for so long to get something like that to work. thank you :upside_down_face: